mk/docs.mk

```diff
@@ -273,11 +273,13 @@ compiler-docs: $(COMPILER_DOC_TARGETS)
 trpl: doc/book/index.html
 
 doc/book/index.html: $(RUSTBOOK_EXE) $(wildcard $(S)/src/doc/trpl/*.md) | doc/
+	@$(call E, rustbook: $@)
```

Comment:
Ah yes I figured I'd just clean up the output here to be a little more consistent with the rest of the build (now that it doesn't print the chapters to stdout by default)